File: oct.cpp
#include <iostream>
#include <sys/time.h>

//command to compile: gcc oct.cpp -o oct -lstdc++

int main () {
        unsigned long int cur = time(NULL);
        unsigned long int past = cur - 1633060800;
        unsigned long int days = past / 86400;
        std::cout << "its october " << days << "\n";
        return 0;
}

File: oct.cs
using System;

class oct {         
        static void Main(string[] args)
        {
                long seconds = (long)DateTime.UtcNow.Subtract(DateTime.UnixEpoch).TotalSeconds;
                long past = (long)seconds - 1633060800;
                long days = (long)past / 86400;
                Console.WriteLine("its october {0}", days);
        }
}
